packages/hasher/src/TargetHasher.ts

```diff
@@ -185,6 +188,11 @@ export class TargetHasher {
     ]);
 
     await this.initializedPromise;
+
+    if (this.logger !== undefined) {
+      const globalInputsHash = hashStrings(Object.values(this.globalInputsHash ?? {}));
+      this.logger.silly(`Global inputs hash: ${globalInputsHash}`);
```

Comment:
We are logging input hashes at the "verbose" level, thinking we should do it at that level.